sklearn/utils/sparsefuncs.py

```diff
@@ -25,6 +26,49 @@ def _raise_error_wrong_axis(axis):
         raise ValueError(
             "Unknown axis value: %d. Use 0 for rows, or 1 for columns" % axis)
 
+def csr_csc_min_axis0(X):
```

Comment:
I'll agree to working on the backport solution, what would be the necessary steps to get this started?

(I'll also try to reduce the amount of code duplication.)